templates.py

```diff
@@ -0,0 +1,14 @@
+
+NEW_CTF = """New CTF!

```

Comment:
maybe consider changing anonymous template parameters (`{}, starts at {}`) to named ones (`{ctf_name}, starts at {ctf_start}`? Makes templates a bit more readable, and template substitution a bit less volatile (`NEW_CTF.format(ctf_name="bork", ctf_start="always")`)